### PR TITLE
Update social-media.md

### PR DESCRIPTION
**Pull request by Phyloiota**
Created page and links...This should be all officially by IOTA Foundation run channels

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: docs/participate/the-community/social-media.md [hunk 1]
```diff
@@ -1,34 +1,70 @@
 ---
 title: Social Media
-description: IOTA has a threiving and active community on all popular social media applications. Find the IOTA community on your prefered medium.
+description: IOTA has a threiving and active community on all popular social media applications. Find the IOTA Foundation and the community here.
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:3]:
```suggestion
description: IOTA has a thriving and active community on all popular social media applications. Find the IOTA Foundation and the community here.
```

File: docs/participate/the-community/social-media.md [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
-# Heading 1 (Page Title)
+# IOTA Social Media
 
-content - introduction to the topic
+The IOTA project has established a social media presence on all popular social media channels. These are the presences handled by the IOTA Foundation:
 
+## Discord
 
-## Heading 2 (Sub topic 1)
+This is without doubt the most active community and with nearly 50.000 members a great place for everyone that wants to learn more about IOTA and get to know the people involved. A lot of IOTA Foundation Team members and also the Founders and members of the Board are active on Discord and take part in the community discussions and projects there. Every aspect of the project is covered here in separate channels and you can find the whole range of this amazing community from serious mathematics and cryptographic topic up to memes and Fun channels.
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:13]:
```suggestion
This is without doubt the most active community and with nearly 50,000 members a great place for everyone that wants to learn more about IOTA and get to know the people involved. A lot of IOTA Foundation Team members and also the Founders and members of the Board are active on Discord and take part in the community discussions and projects there. Every aspect of the project is covered here in separate channels and you can find the whole range of this amazing community from serious mathematics and cryptographic topic up to memes and Fun channels. 
```

File: docs/participate/the-community/social-media.md [hunk 3]
```diff
@@ -1,34 +1,70 @@
...
-content 
+## Twitter
 
-## Heading 2 (Sub topic 2)
+The official IOTA Foundation Twitter Account is this one here, which will provide the newest developments, announcements, and information regarding the IOTA Project:
 
-content
+[IOTA Foundation Twitter](https://twitter.com/iota)
 
+The engineering Team of the IOTA Foundation also runs its own Twitter account for Technologie updates:
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:24]: Mabe it's just me, but could you rephrase this sentence? Technologie updates doesn't sound right to me 🤔

File: docs/participate/the-community/social-media.md [hunk 4]
```diff
@@ -1,34 +1,70 @@
...
-- **[a link](https://linkgoes.here)** - A description for the link
-- **[a link](https://linkgoes.here)** - A description for the link
-- **[a link](https://linkgoes.here)** - A description for the link
+[IOTA Foundation](https://www.youtube.com/c/iotafoundation)
+
+We hope you enjoy the content and subscribe.
+
+## Reddit
+
+The IOTA community on Reddit meets in its own subreddit to discuss everything related to the project:
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:41]:
```suggestion
The IOTA community on Reddit meets in its own Subreddit to discuss everything related to the project:
```

File: docs/participate/the-community/social-media.md [hunk 5]
```diff
@@ -1,34 +1,70 @@
...
+[IOTA Stack Exchange](https://iota.stackexchange.com/)
+
+
+## Instagram
+
+Follow the IOTA Foundation on Instagram on the official account:
+
+[IOTA Instagram](https://www.instagram.com/iotafoundation)
+
+## Linkedin
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:58]:
```suggestion
## LinkedIn
```

File: docs/participate/the-community/social-media.md [hunk 6]
```diff
@@ -1,34 +1,70 @@
...
+
+The company account of IOTA Foundation can be found here. See recent job postings and engage on business level:
+
+[IOTA Foundation Linkedin](https://www.linkedin.com/company/iotafoundation/)
```
Comment (Dr-Electron) [inline docs/participate/the-community/social-media.md:62]:
```suggestion
[IOTA Foundation LinkedIn](https://www.linkedin.com/company/iotafoundation/)
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: docs/participate/the-community/social-media.md (hunk 3 above)
Comment (Phyloiota) [reply docs/participate/the-community/social-media.md:24]: Yeah good point - its "denglishing" too much - ill rewrite that

**Review by Dr-Electron — APPROVED** [review]
